vortex-file/src/read/layouts/test_read.rs

```diff
@@ -17,9 +16,7 @@ pub fn layout_splits(layout: &mut dyn LayoutReader, length: usize) -> Vec<RowMas
     splits
         .into_iter()
         .tuple_windows::<(usize, usize)>()
-        .map(|(begin, end)| unsafe {
-            RowMask::new_unchecked(Bitmap::from_range(begin as u32..end as u32), 0, end)
-        })
```

Comment:
this logic was incorrect (didn't match real code) which then masked the bug that I fixed. Need to figure out how to customize regular reader so we don't need this code